CONTRIBUTE.md

```diff
@@ -0,0 +1,104 @@
+Contributing to a11y.css
+========================
+
+You can contribute to a11y.css by adding a language or tackling a known issue listed below.
+
+*PS: CSS best practices do not belong to this CSS file. Some selectors are ugly as hell, but we need them in order to select what we need to select.*
+
+## Add languages
+
+a11y.css currently exists in both English and French: you'll find both files in the [css](https://github.com/ffoodd/a11y.css/tree/master/css) folder. If you want to contribute and add a new language, it is very easy to do:
+
+1. Add the name of the language in [this list](https://github.com/ffoodd/a11y.css/blob/master/sass/utils/_mixins.scss#L10);
+2. Copy [`_a11y-en.scss`](https://github.com/ffoodd/a11y.css/blob/master/sass/a11y-en.scss), rename it `_a11y-<your-language>.scss` and change the value passed to the `set-locale` mixin;
+3. Update [this map](https://github.com/ffoodd/a11y.css/blob/master/sass/utils/_variables.scss#L10) with your translations.
+4. Update [this map](https://github.com/ffoodd/a11y.css/blob/master/sass/utils/_variables.scss#L241) to translate theme names.
+5. Run `sass --update sass:css --style=compressed` to generate the new CSS file.
+
+Done!
+
+## Edge cases and known issues
```

Comment:
Maybe it's best to not list any particular issue here and link to issues on github directly ?